### PR TITLE
Added mandatory labels

### PR DESCRIPTION
**Pull request by eliskasl**
@mfojtik @brenton, please have a look

### CODE REVIEW

File: 9.2/Dockerfile.rhel7 [hunk 1]
```diff
@@ -16,7 +16,11 @@ ENV POSTGRESQL_VERSION=9.2
 LABEL k8s.io/description="PostgreSQL is an advanced Object-Relational database management system" \
       k8s.io/display-name="PostgreSQL 9.2" \
       openshift.io/expose-services="5432:postgresql" \
-      openshift.io/tags="database,postgresql,postgresql92"
+      openshift.io/tags="database,postgresql,postgresql92" \
+      LABEL BZComponent="postgresql92" \
```
Comment (mfojtik) [inline 9.2/Dockerfile.rhel7]: nuke the `LABEL` instruction (the LABEL support multi-line assignment)

**Comment by mfojtik** [comment]
@eliskasl please add comment to Dockerfile explaining why we need this labels.

**Comment by mfojtik** [comment]
@eliskasl actually I think it would be better to move RCM labels into new layer (we are going to squash that anyway). But it makes it more readable in sense that we know who owns what labels. Also you can add comment on top of that layer, saying that these labels belongs to RCM.

**Comment by mfojtik** [comment]
cc @bparees

**Comment by mfojtik** [comment]
LGTM

**Comment by bparees** [comment]
same question here on whether openshift3 is the right value for the name field, vs openshift.

**Comment by brenton** [comment]
As best I can understand our official naming policy I believe `/openshift3` is the most correct.  For example we were asked to change from `/openshift` to `/openshift3_beta` for the beta period.

The `BZComponent` label is new to me.  I thought it was simply `Component`.

**Comment by bparees** [comment]
ok, that has an impact on our templates:
https://github.com/openshift/origin/blob/master/examples/image-streams/image-streams-rhel7.json#L14
(I guess they needed to be updated anyway)

so if we're sure it's going to be openshift3 i'll start updating them.

**Comment by brenton** [comment]
thx ben.  I think it's safe to update those templates now.

**Comment by mfojtik** [comment]
@brenton @bparees we still use Docker Hub centos images in those templates so I think we should also get the "openshift3" namespace in Docker Hub.

**Comment by eliskasl** [comment]
BZComponent changed to Component. Thanks Brenton!

**Pull request merged**